src/webui/webui.qrc

```diff
@@ -1,49 +1,48 @@
 <RCC>
     <qresource prefix="/">
+        <file>www/private/about.html</file>
```

Comment:
I would prefer leave the order as it was (aka "smart" alphabetical order, i.e. the folders appear first, and the files after the folders).

Reply:
Is there a tool for it? (web app?)
Currently I use http://sortmylist.com/ and this was its result.

Reply:
My tool is my hands.
BTW, the list was sorted this way before.

Reply:
I don't object for the sorting order, but without a handy tool it is troublesome get it done quickly and also error prone.

Reply:
Also worth noting that GNU sort: `sort --ignore-case webui.qrc` (with manually fixing the headers) produces the same output as sortmylist.

Reply:
Okay, forget it. It just seems to me more readable/ordered when folders go first (file managers usually show it this way). But in the case of resources, I don't care how it is ordered there, because I rarely look there. I just want the order obeyed some rule, otherwise everyone will start to rearrange it to your liking (without any benefit).